### PR TITLE
Fix #57

### LINKED ISSUE #57: Can't scroll to bottom on dbMon example in Chrome
When scrolled down to the bottom-most , scroll stuck , can't get go the bottom.
screenshot_20170426_181913

### PR DESCRIPTION
**Pull request by runspired**
Fixes #57

### CODE REVIEW

**Comment by runspired** [comment]
Tests are red but between both runs all versions were green, we really need to figure out this flakiness :'(

File: addon/-private/data-view/radar/radar.js [hunk 1]
```diff
@@ -289,6 +289,21 @@ export default class Radar {
     }
   }
 
+  _updatePoolContents() {
```
Comment (pzuraq) [inline addon/-private/data-view/radar/radar.js]: We can remove this function. `_updateVirtualComponents` will always update all of the items in the pool, we just need to make sure that `itemDelta` is zero if we don't want it to jump around (like it was in the dbmon example).

Comment (runspired) [inline addon/-private/data-view/radar/radar.js]: see later fix :P

File: addon/-private/data-view/radar/radar.js [hunk 2]
```diff
@@ -307,12 +322,16 @@ export default class Radar {
     this.scheduleUpdate();
   }
 
-  resetItems(items) {
+  updateItems(items, isReset = false) {
```
Comment (pzuraq) [inline addon/-private/data-view/radar/radar.js]: Renaming this function has implications for DynamicRadar, it overrides the `resetItems` function and calls super. We either need to roll that logic into here somehow, or we need to rethink the reset logic.

Comment (runspired) [inline addon/-private/data-view/radar/radar.js]: I fixed it there too

File: addon/-private/data-view/radar/radar.js (hunk 1 above)
Comment (pzuraq) [inline addon/-private/data-view/radar/radar.js]: 👍

**Pull request merged**